packages/calypso-e2e/src/data-helper.ts

```diff
@@ -289,3 +289,29 @@ export function createSuiteTitle( title: string ): string {
 
 	return parts.join( ' ' );
 }
+
+/**
+ * Parses a WPCOM site host name (without the scheme) from a given site-specific page URL.
+ * For example:
+ *  - testsite.wordpress.com/wp-admin/post --> testsite.wordpress.com
```

Comment:
I think it's good that we're handling both URL formats here! I'm curious though: considering this is now only called from post page - which should be running "from" Calypso - did you actually already stumble upon a scenario where it was called while the top-level site site had a `wp-admin` URL?